package.json

```diff
@@ -3,10 +3,14 @@
   "version": "1.0.0-beta.1",
   "description": "Discover, Install, Uninstall, Submit, Remove Adobe App Builder templates",
   "dependencies": {
+    "@adobe/aio-cli-lib-app-config": "^0.2.2",
+    "@adobe/aio-lib-console": "^3.2.0",
+    "@adobe/aio-lib-console-project-installation": "github:adobe/aio-lib-console-project-installation#DEVX-2298",
     "@adobe/aio-lib-core-config": "^2.0.0",
     "@adobe/aio-lib-core-logging": "^1.1.0",
     "@adobe/aio-lib-ims": "^4.3.0",
     "@adobe/aio-lib-templates": "^1.1.0",
+    "@adobe/generator-app-asset-compute": "^0.2.2",
```

Comment:
Please, remove. It is from the test installations I believe.

Reply:
✅ Done